airflow/providers/jira/operators/jira.py

```diff
@@ -45,21 +46,21 @@ class JiraOperator(BaseOperator):
 
     @apply_defaults
     def __init__(self,
-                 jira_conn_id='jira_default',
-                 jira_method=None,
-                 jira_method_args=None,
-                 result_processor=None,
-                 get_jira_resource_method=None,
+                 jira_conn_id: str = 'jira_default',
+                 jira_method: Any = None,
```

Comment:
```suggestion
                 jira_method: Optional[str] = None,
```

Reply:
Made this change but requires type hint on line 83 because getattr doesn't like getting a None passed as argument

Reply:
This parameter seems to be required. If the user does not pass this argument, they will always have an error.

Reply:
Thanks...changed parameter to required